docs/conf/eu/2016/news/schedule.md

```diff
@@ -0,0 +1,85 @@
+:template: 2016/eu.html
+
+Announcing the conference schedule
+=====================================================================================
+
+Write the Docs is just over three weeks away now! We hope you are all as excited as we are for the conference.
+
+Tickets are **almost sold out**, so make sure to [purchase a ticket][ticket] now, if you haven't already. We'll be closing ticket sales on the *5th September* to leave us enough time to print shirts and badges -- don't miss out!
```

Comment:
Sept 5 seems a bit early -- perhaps we should just say if you don't get a ticket by then, you won't have a badge? Or give a vague date "second week of Sept"